resources/[system]/scoreboard/client.lua

```diff
@@ -0,0 +1,118 @@
+RegisterKeyMapping("+scoreboard", "Open the scoreboard.", "keyboard", "z")
+
+RegisterCommand("+scoreboard", function(source, args, rawcommand)
+  if not isShowing then
+    TriggerServerEvent("scoreboard:getPlayers")
```

Comment:
Latent events and some sort of rate limiting (now you can just hit the button a lot and it'll request a _lot_ of updates)?